### ISSUE TITLE
I can compress the editor to the point where no icons are visible at all

### ISSUE THREAD

**Issue by Pomax** [mozilla/thimble.mozilla.org#2140]
There should probably be a width-cap or minimal width for the editor... being able to reduce it to this makes no sense, at that point the preview should probably just become full screen:

![screenshot 47](https://cloud.githubusercontent.com/assets/177243/26085351/674cafa0-3998-11e7-8171-36f740ab7caf.png)

**Comment by humphd** [comment]
Good idea, and connected to https://github.com/mozilla/thimble.mozilla.org/issues/1902.  We can probably do both at once.

**Comment by humphd** [comment]
We should consider adding a minimum width on this, and/or auto-closing (e.g., hiding it) when it goes past a certain size.

@flukeout can you give your opinion on how you want this accomplished?

**Comment by mmBabol** [comment]
Hello! I'm wondering if I can take this bug? I'm a beginner wanting to start in the Open Source world. 
What would be a suitable min-width?

**Comment by humphd** [comment]
@mmBabol all yours, let us know what help you need.

**Comment by ashish1500616** [comment]
Hi @humphd  its seems that this issue has not been resolved and no one is working on it, can I work on this issue?

**Comment by humphd** [comment]
@ashish1500616 fine with me if @mmBabol is OK with that.

**Comment by ashish1500616** [comment]
@mmBabol Is it ok with you If I take up this issue.

**Comment by gideonthomas** [comment]
@ashish1500616, if you're still interested in this issue, feel free to take it over since we haven't had a response from the previous contributor.
